### DISCUSSION TITLE
Changing the web login screen

### DISCUSSION THREAD

**Discussion by IITYWYBMAB** [gravitational/teleport#36161] [Ideas]
Hi All,

I've just got the community Teleport up and running and it's a great product.  I can see myself using this quite a bit.  My only concern at this point is being that it's an exposed endpoint I would like to obscure the login page so it's not declaring to the world what CVE's may be available for it.  Is there a way to change the login screen to something more minimal that obscures what's being logged into?  I have other ideas as well, yet this seems to be potentially the easiest..

Thanks!

Keith

**Comment by zmb3** [comment]
No, there is no way to change the login screen. 

Protection against CVEs is best acquired via regular and timely patching, not via obscurity.

**Comment by IITYWYBMAB** [comment]
Security via obscurity is a thing, particularly when a there's an undisclosed vulnerability.  I'll put it behind a WAF, that should at least be helpful.  I've also switched up the port to a random high one, that should also be helpful.  Likewise, I am thinking it could be a nice feature, either, “custom login page”, or just a checkbox for “minimal login page”.   I try to obscure most of my public access to my internal net in addition to other security, and this would be a nice add-on.  I'm not a high-value target, yet I assume many of your customers are.
